src/common/snippets/src/op/subgraph.cpp

```diff
@@ -232,6 +233,7 @@ void snippets::op::Subgraph::convert_to_snippet_dialect() {
     manager.register_pass<snippets::pass::LoadMoveBroadcastToBroadcastLoad>();
     manager.register_pass<snippets::pass::ReplaceLoadsWithScalarLoads>();
     manager.register_pass<snippets::pass::ReplaceStoresWithScalarStores>();
+    manager.register_pass<snippets::pass::MulAddToFMA>();
```

Comment:
Conversion to FMA is a plugin-specific optimization, so it shouldn't be included in snippets common pipeline.
Please, pass the optimization to generate from the plugin snippets node.

Reply:
Done, thanks